webdriver-spec.html

```diff
@@ -5047,9 +5052,29 @@
    <dd><p>Toggle <var>element</var>’s <a>selectedness</a> state.
 
    <dt><a><code>option</code> element</a>
-   <dd><p>Set <var>element</var>’s <a>selectedness</a> state to true.
+   <dd>
+     <ol>
+       <li><p>Let <var>currently selected</var> is the <a>container</a> with
+         a <a>selectedness</a> state of true.
+
+       <li><p>Let <var>parent node</var> be the calling
```

Comment:
Missing words. `result`, possibly? And don't we want to focus the container rather than the container's parent? For an OPTION, the container is typically the SELECT element.